admin_manual/configuration_user/user_auth_ldap.rst

```diff
@@ -466,6 +467,84 @@ In migrated Nextcloud installations the old behavior still applies, which is usi
 
   sudo -u www-data php occ config:app:set user_ldap enforce_home_folder_naming_rule --value=1
 
+"$home" Placeholder Field:
+  $home in an external storage configuration will be replaced with the value of the specified attribute
+
+.. _LDAP_User_Profile_Attributes:
+
+User Profile attributes
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. figure:: ../images/ldap-advanced-4-attributes.png
+   :alt: User Profile Attributes.
+
+After configuring those attributes, the User Profile data will be overwritten with the according data from LDAP.
+The data must be formatted like Nextcloud wants it, otherwise it will be constantly changed and rechanged/removed again.
```

Comment:
This would need to publishing the protected functions sanitizePhone/sanitizeWebsite from AccountManager, because those are not accessible from user land code.

Reply:
@come-nc I refined the code again, for better handling.
The profile data gets updated only, if the LDAP data has changed. I calculate and store a checksum of the profileValues from LDAP. If the checksum matches, the profile update gets skipped. Secondly, the user can now change his profile without it getting overwritten instantly. Thirdly, I write the checksum to cache and skip until the cached value expired.
Hope you like it.